deploy/crds/managed.openshift.io_mustgathers.yaml

```diff
@@ -49,13 +49,24 @@ spec:
                       TODO: Add other useful fields. apiVersion, kind, uid?'
                     type: string
                 type: object
+              internalUser:
+                default: true
+                description: A flag to specify if the upload user provided in the  caseManagementAccountSecret
+                  is a RH internal user. See documentation for further information.
+                type: boolean
```

Comment:
@dustman9000 this allows the user to let us know in the yaml for the CR whether we should treat them as internal hydra users or not. I decided this should default to true because our main user base, CEE engineers, will all be internal users (so if they don't have this field in their CR yaml, it won't cause any issues for them)